src/browser/core_state.ts

```diff
@@ -473,7 +473,32 @@ export function getWindowByUuidName(uuid: string, name: string): Shapes.OpenFinW
     return win && win.openfinWindow;
 }
 
-function getOfWindowByUuidName(uuid: string, name: string): Shapes.Window|false {
+export function getBrowserWindow(identity: Shapes.Identity): Shapes.BrowserWindow|undefined {
+    const { uuid, name } = identity;
+    const wnd: Shapes.Window = getOfWindowByUuidName(uuid, name);
+
+    if (wnd && wnd.openfinWindow) {
+        return wnd.openfinWindow.browserWindow;
+    }
+}
+
+export function getWebContents(identity: Shapes.Identity): webContents|undefined {
+    const browserWindow = getBrowserWindow(identity);
+
+    if (browserWindow) {
+        return browserWindow.webContents;
```

Comment:
Do we need an `isDestroyed` check?

Reply:
Ok, I'll add it 👍

Reply:
[Added to `getBrowserWindow` instead](https://github.com/HadoukenIO/core/pull/423/files#diff-5990126ad71fa4d6d779051f5546f991R505)